src/Assert.php

```diff
@@ -2042,6 +2045,44 @@ public static function isMap(mixed $array, string $message = ''): array
         return $array;
     }
 
+    /**
+     * @param Closure $closure
+     * @param string $message
```

Comment:
No need for this, it matches the code.

Also missing `@psalm-assert` docs.

Reply:
I've attempted this but I'm not familiar with the psalm assert docs. If you could suggest the changes, that would be extremely helpful.